config.yaml

```diff
@@ -1,10 +1,3 @@
-# PLEASE ONLY ENTER YOUR PASSPHRASE BELOW IF YOU ARE GOING TO MINE SOLO!
-# YOU DO NOT NEED TO ENTER YOUR PASSPHRASE FOR POOL MINING.
-# NEVER POST YOUR CONFIG.YAML INCLUDING YOUR PASSPHRASE TO ANY BOARD OR CHAT!
-account_id_to_secret_phrase:          # define accounts and passphrases for solo mining 
-  10282355196851764065: 'glad suffer red during single glow shut slam hill death lust although'          # Bold's example wallet
-  1796535821016683299: 'stand rude those door invite reflection anywhere lace safe hidden fur horrible'  # Johnny's example wallet
-
```

Comment:
Removing this example means nobody will know how to set up solo mining, doesn't it?

Reply:
Nevermind. After consulting via discord, this is not a concern since we want to encourage running a private node for solo mining, and when you do that your miner passphrase is configured in the node, rather than in the miner, and it'll work.